packages/xo-server-test/src/_xoConnection.js

```diff
@@ -278,7 +278,7 @@ afterAll(async () => {
   await xo.close()
   xo = null
 })
-afterEach(() => xo.deleteTempResources())
+afterEach(() => xo.deleteTempResources(), 30e3)
```

Comment:
Can we put it in the config file?

Reply:
Are you OK with `config.xoConnection.afterEachTimeout` ?

Reply:
What about `config.deleteTempResourcesTimeout`?

Reply:
I think it can limit us if we want to add more actions in the `afterEach` because it's not the `deleteTempResources` timeout but the `afterEach` timeout.

Reply:
No, because other actions will go in other `afterEach`s.